datastream-utils/src/main/java/com/linkedin/datastream/common/LogUtils.java

```diff
@@ -62,4 +66,30 @@ public static String logNumberArrayInRange(List<Integer> list) {
       return list.toString();
     }
   }
+
+  /**
+   * Shortening the list of topic-partition mappings by merging partitions of the same topic together. e.g.
+   * topic1-0, topic1-1, topic2-0 -> topic1:[0-1], topic2:[0]
+   * @param list list of strings to generate logging String for
+   * @return compacted String that merges partitions per topic
+   */
+  public static String logOrganizeTopicPartitionMappings(List<String> list) {
+    Map<String, List<Integer>> topicPartitionsV2 = new HashMap<>();
+    try {
+      for (String partition : list) {
+        String[] tokens = partition.split("-");
```

Comment:
can you reuse [code](https://github.com/linkedin/brooklin/blob/d12144a4dad2bb973ca7fcc9c6088c5c46a49172/datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/TopicPartitionUtil.java) to split topic/partition name?

Reply:
Couldn't reuse that function from its original package, as there was a modular circular dependency. So, moved it to common module.